administration/src/bp-modules/stores/StoresController.tsx

```diff
@@ -1,10 +1,25 @@
+import { ButtonGroup } from '@blueprintjs/core'
 import React, { ReactElement } from 'react'
+import { useNavigate } from 'react-router-dom'
+import styled from 'styled-components'
 
-type StoresControllerProps = {}
+import StandaloneCenter from '../StandaloneCenter'
+import CardFormButton from '../cards/CardFormButton'
 
-const StoresController = (props: StoresControllerProps): ReactElement => {
-  // TODO 1475 CSV Store Import Module
-  return <div>Store Import</div>
+const Buttons = styled(ButtonGroup)`
+  width: 400px;
+`
+
+const StoresController = (): ReactElement => {
+  const navigate = useNavigate()
+  return (
+    <StandaloneCenter>
+      <Buttons vertical>
+        <CardFormButton text='Akzeptanzpartner CSV Import' icon='upload' onClick={() => navigate('./import')} />
+        <CardFormButton text='Akzeptanzpartner ansehen und bearbeiten' icon='add' onClick={() => navigate('./add')} />
```

Comment:
why the route is called `add` if it's `view and edit` ?

Reply:
i think all of the three. there is no concrete concept for that.
When the concept is ready we also may provide this feature to the customers
I can also remove this second button, its more a placeholder

Reply:
removed that route, since no decision how it will look like